skrub/tests/test_minhash_encoder.py

```diff
@@ -254,3 +254,36 @@ def test_get_feature_names_out(df_module):
     encoder.fit(X)
     expected_columns = ["col1_0", "col1_1", "col1_2", "col1_3"]
     assert encoder.get_feature_names_out() == expected_columns
+
+
+@pytest.mark.parametrize(
+    "n_components, expected_columns",
+    [
+        (3, ["col_0", "col_1", "col_2"]),  # No padding needed for components < 10
+        (
+            12,
+            [
+                "col_00",
+                "col_01",
+                "col_02",
+                "col_03",
+                "col_04",
+                "col_05",
+                "col_06",
+                "col_07",
+                "col_08",
+                "col_09",
+                "col_10",
+                "col_11",
+            ],
+        ),  # 2-digit padding
+    ],
+)
+def test_zero_padding_in_feature_names_out(df_module, n_components, expected_columns):
+    """Test that ``get_feature_names_out`` returns the correct zero-padded feature names."""
```

Comment:
the only formatting thing left is that the formatter complains this line is too long. it cannot reformat it itself because it is the docstring and it only modifies code, not docstrings nor comments so we have to fix it manually. maybe we can make it a bit shorter like "Check the feature names are zero-padded"